sip-archetype/src/main/resources/archetype-resources/__rootArtifactId__-domain/pom.xml

```diff
@@ -13,10 +13,12 @@
     <name>${artifactId}</name>
 
     <dependencies>
+#if (${useLombok} == "y")
```

Comment:
Let's go a step forward and refactor a bit.
Remove lombok dependency from demo pom as it is inherited from its parent.

Reply:
I agree!